webapp/Connector/src/app/model/MAb.js

```diff
@@ -0,0 +1,56 @@
+Ext.define('Connector.app.model.MAb', {
+
+    extend : 'Ext.data.Model',
+
+    idProperty: 'mab_mix_name_std',
+
+    labelProperty: 'mab_mix_name_std',
+
+    resolvableField: 'mab_mix_name_std',
+
+    dataAvailabilityField: 'studies_with_data',
+
+    fields: [
+        {name: 'mab_mix_id'},
+        {name: 'mab_mix_name_std'},
+        {name: 'mab_mix_type'},
+        {name: 'mab_mix_lanlid'},
+        {name: 'mab_mix_lanlid_link'},
+        {name: 'mab_mix_name_other'},
+        {name: 'other_labels'},
+
+        {name: 'has_data'},
+
+        {name: 'mabs', convert : function(value) {
```

Comment:
Could declare a converter function that all of the model fields could use throughout the app that are attempting to ensure Array. There may be other convert methods that could be shared as well?

```js
function asArray(value) {
    return Ext.isArray(value) ? value : [];
}

// usage
{name: 'some_field', convert: asArray}
```